packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -139,6 +115,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
     this._isFocusingPreviousElement = false;
     this._isScrollIdle = true;
+    this._contextualMenuChildrenAs = props.contextualMenuChildrenAs || ContextualMenuItemChildren;
```

Comment:
The reason I used this was because typescript didn't understand that props.contextualMenuChildrenAs was being set by defaultProps. It was then mad when I tried to instantiate the React component because it thought it might potentially be undefined.

I guess I could use default props and add safety checks everywhere that accesses contextual MenuChildrenAs. Do you know any other way?

Reply:
There really isn't a good way to get around that yet. Instead of safety checks though I think you can get by with doing something like `let {contextualMenuChildrenAs! as RenderFunction} = this.props.`

Reply:
Maybe not. Let me double check

Reply:
Okay, new plan, it's not ideal but doing like `let { contextualMenuChildrenAs: RenderFunction = DefaultRender } = this.props` is the cleanest way and keeps things in props for now.

Reply:
Will do.

This syntax is wild, I did not know you could destructure and alias at the same time. I would've guessed the colon would've been 'as'.